dcf/models.py

```diff
@@ -115,3 +115,12 @@ def save(self, *args, **kwargs):
 class Image(models.Model):
     item = models.ForeignKey(Item)
     file = ImageField(_('image'), upload_to='images')
+
+
+class FavoritsInfo(models.Model):
+    customuser = models.ForeignKey(CustomUser)
+    item = models.ForeignKey(Item)
+    liked = models.DateTimeField(null=True, blank=True, auto_now_add=True)
+
+    class Meta:
+        auto_created = True
```

Comment:
I'm sure that Meta option works not as you expected. You need to have `DateTimeField` with `auto_now_add=True` option instead.